app/code/Magento/Catalog/Model/CategoryRepository.php

```diff
@@ -80,6 +80,10 @@ public function save(\Magento\Catalog\Api\Data\CategoryInterface $category)
         $existingData = array_diff_key($existingData, array_flip(['path', 'level', 'parent_id']));
         $existingData['store_id'] = $storeId;
 
+        if (array_key_exists('name', $existingData) && $existingData['name'] === null) {
```

Comment:
Instead of unsetting, please fix a place where name is set to `null`.

Are you sure such request in issue is valid?

Reply:
The request is valid.
See screenshot in issue. This is a real world example. One of our customers wan't to change the the "is_active" flag. It is not possible to change the global value without to specify the name. If you send the same call with a store code it is OK.
Partial updates are supported by Magento.

Reply:
@orlangur I am also not satisfied by this solution but the problem is in the framework.
The "null" value is generated by this method: `\Magento\Framework\Reflection\DataObjectProcessor::buildOutputDataArray`.
Do you have any good idea to handle that?